packages/entities/entities-shared/src/components/entity-base-config-card/EntityBaseConfigCard.vue

```diff
@@ -17,7 +17,38 @@
     <template #actions>
       <div class="config-card-actions">
         <slot name="actions" />
-        <KClipboardProvider v-slot="{ copyToClipboard }">
+        <KLabel
+          v-if="konnectJsonYamlEnabled"
+          class="config-format-select-label"
+          data-testid="config-format-select-label"
+        >
+          {{ label }}
+        </KLabel>
+        <KSelect
+          v-if="konnectJsonYamlEnabled"
+          appearance="select"
+          :items="configFormatItems"
+          @change="handleChange"
+        />
+
+        <KButton
+          v-if="showBookButton"
+          appearance="btn-link"
+          class="book-icon"
+          @click="handleDocumentationClick"
+        >
+          <template #icon>
+            <KIcon
+              color="#0044f4"
+              icon="book"
+            />
```

Comment:
suggestion: instead of using `KIcon` can you please import the icon from `@kong/icons`